src/UI/class-parsely-sites-table.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Parsely\UI;
+
+use Parsely\Parsely;
+use WP_List_Table;
+
+if ( ! class_exists( 'WP_List_Table' ) ) {
+	require_once ABSPATH . 'wp-admin/includes/class-wp-list-table.php';
+}
+
+
+final class Parsely_Sites_Table extends WP_List_Table {
+	/**
+	 * Instance of Parsely class.
+	 *
+	 * @var Parsely
+	 */
+	private $parsely;
+
+	/**
+	 * Constructor.
+	 *
+	 * @param Parsely $parsely Instance of Parsely class.
+	 */
+	public function __construct( Parsely $parsely ) {
+		parent::__construct();
+
+		$this->parsely = $parsely;
+	}
+
+	/**
+	 * @return array
+	 */
+	public function get_columns(): array {
+		return array(
+			'blog_id'     => __( 'Blog ID', 'wp-parsely' ),
+			'blog_name'   => __( 'Blog Name', 'wp-parsely' ),
+			'api_key_set' => __( 'Status', 'wp-parsely' ),
+			'settings'    => __( 'Settings', 'wp-admin' ),
+		);
+	}
+
+	/**
+	 * @return void
+	 */
+	public function prepare_items(): void {
+		$this->_column_headers = $this->get_column_info();
+		$this->items           = $this->fetch_table_data();
+	}
+
+	/**
+	 * @param $item
+	 * @param $column_name
+	 *
+	 * @return string
+	 */
+	public function column_default( $item, $column_name ): string {
+		if ( $column_name == 'settings' ) {
+			return '<a href="' . $item[ $column_name ] . '">Parse.ly Site Settings</a>';
+		}
+
+		return strval( $item[ $column_name ] );
+	}
+
+	/**
+	 * @return array
+	 */
+	private function fetch_table_data(): array {
+		$parsely_network_sites = array();
+		foreach ( get_sites() as $site ) {
```

Comment:
I think `get_sites` will cap at 100 results per:
https://github.com/WordPress/wordpress-develop/blob/5.8.1/src/wp-includes/class-wp-site-query.php#L176

So, we should probably either paginate or add an indication somewhere that only the first 100 results will be shown.  I think the latter is fine.  If someone is running that many subsites, they're likely going to be reaching for a scripting approach to management.

Reply:
I see _number_ is a parameter, so we could hypothetically bump that number up. I would expect a performance drop if that number is very big, though.

I like your messaging proposal. We can implement pagination later if we see a lot of demand for it.